deno/payloads/v10/channel.ts

```diff
@@ -1306,13 +1306,29 @@ export enum ComponentType {
 	 */
 	Button,
 	/**
-	 * Select Menu component
+	 * Select menu for picking from defined options
 	 */
-	SelectMenu,
+	StringSelect,
```

Comment:
Is this not `TextSelect`?

Reply:
no, docs were fixed in https://github.com/discord/discord-api-docs/pull/5544